__tests__/app.test.js

```diff
@@ -96,10 +96,36 @@ describe('GET /api/articles', () => {
             .get('/api/articles')
             .expect(200)
             .then((response) => {
-                const articles = response.body.articles.rows
+                const {articles} = response.body
                 expect(articles).toBeSortedBy('created_at', { descending: false })
             })
     })
+    it('should accept a topic query which filters the articles by the given topic', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then((response)=>{
+            const {articles} = response.body
+            expect(articles).toHaveLength(1)
```

Comment:
Asserting the whole response would be more robust rather than just a length of 1 👀